ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.scss

```diff
@@ -0,0 +1,52 @@
+$border-color: #EBEBEB;
+$icon-width: 24px;
+$icon-margin-right: 2rem;
+$question-padding: 1.5rem;
+
+.faq-block__icon {
+    font-size: 30px;
+    margin-right: $icon-margin-right;
+}
+
+.faq-block__question {
+    margin: 0;
+}
+
+.faq-block__text {
+    white-space: pre-wrap;
+}
+
+.mat-expansion-panel {
+    border-top: 2px solid $border-color;
+    font-family: Untitled-Sans-Regular;
```

Comment:
I'd recommend to set this font as a default for the body tag. You can check how it's done for the other studies

Reply:
Good point. Fixed at ee510c5723c7990f01eb018fab32b3622424d9a9